src/prefect/agent.py

```diff
@@ -94,31 +91,31 @@ async def get_and_submit_flow_runs(self) -> List[FlowRun]:
             seconds=self.prefetch_seconds or PREFECT_AGENT_PREFETCH_SECONDS.value()
         )
 
-        # Use the work queue id or load one from the name
-        work_queue_id = self.work_queue_id or await self.work_queue_id_from_name()
-        if not work_queue_id:
-            return []
+        submittable_runs = []
 
-        try:
-            submittable_runs = await self.client.get_runs_in_work_queue(
-                id=work_queue_id, limit=10, scheduled_before=before
-            )
-        except httpx.HTTPStatusError as exc:
-            if exc.response.status_code == status.HTTP_404_NOT_FOUND:
-                raise ValueError(
-                    f"No work queue found with id '{work_queue_id}'"
-                ) from None
-            else:
-                raise
+        # load runs from each work queue
+        for work_queue in await self.get_work_queues():
 
-        # Check for a paused work queue for display purposes
-        if not submittable_runs:
-            work_queue = await self.client.read_work_queue(work_queue_id)
+            # print a nice message if the work queue is paused
             if work_queue.is_paused:
                 self.logger.info(
                     f"Work queue {work_queue.name!r} ({work_queue.id}) is paused."
                 )
 
+            else:
+                try:
+                    queue_runs = await self.client.get_runs_in_work_queue(
+                        id=work_queue.id, limit=10, scheduled_before=before
+                    )
+                    submittable_runs.extend(queue_runs)
+                except httpx.HTTPStatusError as exc:
+                    if exc.response.status_code == status.HTTP_404_NOT_FOUND:
+                        self.logger.error(
+                            f"Work queue {work_queue.name!r} ({work_queue.id}) not found."
+                        )
+                    else:
+                        self.logger.error(exc)
```

Comment:
Do we want a traceback?
```suggestion
                        self.logger.exception(exc)
```

Reply:
Perhaps not, perhaps the traceback is only necessary for unexpected exception types.